### PR TITLE
flasharch.com

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [X] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [X] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/137611
 - https://flasharch.com/archive/play/715f06e9c0241b3d2451d60e41b51e91

### Add your comment and screenshots

- https://www.similarweb.com/website/flasharch.com/#traffic

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -791,6 +791,8 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/137611
+flasharch.com#@#.ad_area_wrap
```
Comment (Yuki2718) [inline BaseFilter/sections/antiadblock.txt:795]: Where the rule came from? I can't even reproduce the issue.

Comment (Yuki2718) [reply BaseFilter/sections/antiadblock.txt:795]: Okay, it's in French list. So the rule should be added to French. @smed79

File: BaseFilter/sections/css_extended.txt [hunk 2]
```diff
@@ -1943,6 +1943,7 @@ indiastudychannel.com#?#.main > div.row:has(> div.col-md-12 > div.item > div.row
 indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div#HP-gpt-passback)
 indiastudychannel.com#?##content_rightbar > div.roundedBlock:has(> div[id^="div-gpt-ad-"])
 !
+flasharch.com#?#div[class*="MuiGrid-grid-lg-"] ~ div[class*="css-"]:has(.adsbygoogle)
```
Comment (Yuki2718) [inline BaseFilter/sections/css_extended.txt:1946]: Looks bit wide.

Comment (piquark6046) [reply BaseFilter/sections/css_extended.txt:1946]: Is there other leftover?

Comment (Yuki2718) [reply BaseFilter/sections/css_extended.txt:1946]: I mean may possibly cause false positive.

Comment (piquark6046) [reply BaseFilter/sections/css_extended.txt:1946]: OK. I will change it.
Confirmed: https://flasharch.com/boards/5/read/749

**Comment by Yuki2718** [comment]
IMO internal tree of `:has()` should be specified.

File: BaseFilter/sections/antiadblock.txt (hunk 1 above)
Comment (Yuki2718) [reply BaseFilter/sections/antiadblock.txt:795]: You have to move the rule to French list. It's not a Base issue.

**Comment by piquark6046** [comment]
@Yuki2718 On AdGuard and uBO, an internal tree (`>`) does not necessary.
Did you mean that it is required to prevent a false positive?

File: BaseFilter/sections/antiadblock.txt (hunk 1 above)
Comment (piquark6046) [reply BaseFilter/sections/antiadblock.txt:795]: Moving ...

Comment (piquark6046) [reply BaseFilter/sections/antiadblock.txt:795]: Moved. https://github.com/easylist/listefr/pull/110

**Review by zloyden — APPROVED** [review]

**Comment by Yuki2718** [comment]
> Did you mean that it is required to prevent a false positive?

Yes, I thought so. But now @zloyden merged.

**Pull request merged**